src/main/webapp/javascript/user-page.js

```diff
@@ -2,29 +2,32 @@
 const NO_PLANNER_RECIPES = "You have not added any recipes to your planner yet.";
 const NO_COOKBOOK_RECIPES = "You have not added any recipes to your cookbook yet.";
 const NO_USER_RECIPES = "You have not uploaded any recipes yet.";
+const USER_RECIPES_DIV = "user-recipes";
 
 // loads all the recipes when the recipe loads
 async function loadRecipes() {
   loadUserRecipes();
+  setUpDivWithNoRecipes("planner-recipes", NO_PLANNER_RECIPES);
+  setUpDivWithNoRecipes("cookbook-recipes", NO_COOKBOOK_RECIPES);
```

Comment:
maybe save "planner-recipes" and "cookbook-recipes" as constants? To be consistent with USER_RECIPES_DIV, and also so they're easier to reuse if you needed to reuse them anywhere